webapp/src/App.tsx

```diff
@@ -17,6 +16,8 @@ import { ObjectsOverlay } from "./components/ObjectsOverlay";
 import { VideoFeed } from "./components/VideoFeed";
 import { PanTilt } from "./components/PanTilt";
 
+let hasConnected = false;
```

Comment:
monitorHubState should be and is already used as a singleton.

Reply:
doh!  except when running tests which startup and tear down App repeatedly. fixing....